From c657c6cbac26fd178778194dfc6d438bac8f2227 Mon Sep 17 00:00:00 2001
From: Alexander Alekhin <alexander.a.alekhin@gmail.com>
Date: Wed, 18 Sep 2019 15:18:09 +0000
Subject: [PATCH] cmake: use 'long long' for atomic check

---
 cmake/checks/atomic_check.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/checks/atomic_check.cpp b/cmake/checks/atomic_check.cpp
index c9770a0bfaa..8ba13b2d69b 100644
--- a/cmake/checks/atomic_check.cpp
+++ b/cmake/checks/atomic_check.cpp
@@ -2,7 +2,7 @@
 
 static int test()
 {
-    std::atomic<int> x;
+    std::atomic<long long> x;
     return x;
 }
 
